mantidimaging/core/utility/unit_conversion.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2024 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+from __future__ import annotations
+
+import numpy as np
+
+
+class UnitConversion:
```

Comment:
Would be good to have some tests for this class.

Reply:
I have changed the UnitConversion class a bit in PR#2159 so I think that it would be better to write the tests for that PR instead